src/pages/patterns/filters.mdx

```diff
@@ -0,0 +1,383 @@
+![Filter pattern basic single select 3](figma:filter-usage-basic-singleselect-3.png)
+![Filter pattern basic single select 4](figma:filter-usage-basic-singleselect-4.png)
+
+#### Multiple selections
+
+Multiple selection basic filters use multiple selections within a category to filter data.
+The Menu remains open as the user makes their selections.
+The selections are surfaced as Tags with a Basic Button to clear all filters.
+
+![Filter pattern basic multiple select 1](figma:filter-usage-basic-multipleselect-1.png)
+![Filter pattern basic multiple select 2](figma:filter-usage-basic-multipleselect-2.png)
+
+#### Nested menu
+
+A [Nested menu](/components/menu) can be used to organize subcategories of data that can be used to filter a dataset.
+
+![Filter pattern basic nested menu 1](figma:filter-usage-basic-nestedmenu-1.png)
+![Filter pattern basic nested menu 2](figma:filter-usage-basic-nestedmenu-2.png)
+![Filter pattern basic nested menu 3](figma:filter-usage-basic-nestedmenu-3.png)
+
+### Advanced filtering
+
+Advanced Filters allow users to set and configure multiple filters on a dataset.
+
+#### Tooltip modal
+
+A Tooltip modal allows users to filter a dataset through one category with many options,
+and allows the user to search for specific options.
+They can also be part of a more complex flow.
+
+![Filter pattern advanced tooltip modal](figma:filter-usage-advanced-tooltip-modal.png)
+
+#### Sheet
+
+A Sheet is used when the dataset is filtered live as the user makes their selections.
+Selections are surfaced as a Tag as the user sets each filter.
+
+![Filter pattern advanced sheet 1](figma:filter-usage-advanced-sheet-1.png)
+![Filter pattern advanced sheet 2](figma:filter-usage-advanced-sheet-2.png)
+![Filter pattern advanced sheet 3](figma:filter-usage-advanced-sheet-3.png)
+![Filter pattern advanced sheet 4](figma:filter-usage-advanced-sheet-4.png)
+![Filter pattern advanced sheet 5](figma:filter-usage-advanced-sheet-5.png)
+![Filter pattern advanced sheet 6](figma:filter-usage-advanced-sheet-6.png)
+
+#### Drawer
+
+A Drawer is used to batch apply multiple filters at once. Selections are surfaced as a Tag
+after the user applies their filters and closes the Drawer.
+
+![Filter pattern advanced drawer 1](figma:filter-usage-advanced-drawer-1.png)
+![Filter pattern advanced drawer 2](figma:filter-usage-advanced-drawer-2.png)
+![Filter pattern advanced drawer 3](figma:filter-usage-advanced-drawer-3.png)
+
+## Behavior
+
+### Bulk actions
+
+When items are selected in a table and a filter is applied, the Filter overrides the selections and deselects them.
+
+![Filter pattern behavior bulk actions 1](figma:filter-behavior-bulk-actions-1.png)
+![Filter pattern behavior bulk actions 2](figma:filter-behavior-bulk-actions-2.png)
+
+## Localization & internationalization
+
+### RTL right-to-left
+
+![Filter pattern localization](figma:filter-localization.png)
```

Comment:
Hmm, not sure what happened. Cut off icon and oddly proportioned input/button split. :) 
<img width="202" alt="Screenshot 2023-08-08 at 9 29 48 AM" src="https://github.com/zendeskgarden/website/assets/3946669/70cd4355-034f-402f-a594-191c95ffe773">

Reply:
That's so weird it was totally normal when I added it but the Figma version is all janky too, I'll update that as well

Reply:
Reminder to check on this again when you have a moment. Still showing as broken.

Reply:
I adjusted the Figma frame and I think I discovered what the issue was, should be fixed now